test/plugins/test_yaml_load_check.py

```diff
@@ -0,0 +1,30 @@
+"""Test yaml_load_check.py"""
+
+from .pylint_test import run_pylint
+
+
+MSG_IDS = 'unsafe-yaml-load'
+
+
+def test_unsafe_yaml_load_warnings():
+    source = """
+    import yaml
+
+    yaml.load('foo.bar')
+    yaml.load()
+    yaml.load('foo.bar', kwargs=**{'a': 'b'})
+    """
+    messages = run_pylint(source, MSG_IDS)
+    expected_msg = 'yaml.load() call is unsafe (unsafe-yaml-load)'
+
+    for actual_msg in messages:
+        assert expected_msg in actual_msg
```

Comment:
No assertion regarding the number of messages?

Reply:
Hah, that would indeed be helpful :)

Reply:
It's also good to tie the messages to specific lines, and to add some pragmas to the test to be sure they work properly for this checker.